src/frontend/src/pages/GanttPage/GanttChartTeamSection.tsx

```diff
@@ -26,47 +36,84 @@ const GanttChartTeamSection = ({
   saveChanges,
   showWorkPackagesMap,
   setShowWorkPackagesMap,
-  teamName,
-  projectTasks,
+  team,
+  projects,
   getNewProjectNumber,
   highlightedChange,
   addProject,
-  addWorkPackage
+  addWorkPackage,
+  getNewWorkPackageNumber,
+  clearEdits,
+  setClearEdits
 }: GanttChartTeamSectionProps) => {
   const theme = useTheme();
-  const [eventChanges, setEventChanges] = useState<EventChange[]>([]);
+  const [ganttChanges, setGanttChanges] = useState<GanttChange[]>([]);
   const [isEditMode, setIsEditMode] = useState(false);
   const [showAddProjectModal, setShowAddProjectModal] = useState(false);
   const id = useId() || 'id';
-
-  const createChange = (change: EventChange) => {
-    setEventChanges([...eventChanges, change]);
+  const [projectsState, setProjectsState] = useState<ProjectPreview[]>([...projects]);
+
+  useEffect(() => {
+    if (clearEdits) {
+      console.log('clearing edits', projects);
+      setProjectsState([...projects]);
+      setClearEdits(false);
+    }
+  }, [clearEdits, projects, setClearEdits]);
```

Comment:
I feel like there could be a better way to do this (the clearEdits/ setClearEdits and having to pass them down and use a useEffect like this). At a minimum you should just pass down `() => setClearEdits(false)` down as resetClearEdits or smthn, since that's all u use it to do in this component

Reply:
I also feel like there could be a better way. If we move the logic from teh gantt page for saving changes into this component it would make it much cleaner the more I think of it